1_fetch/src/download_data.R

```diff
@@ -0,0 +1,8 @@
+download_data <- function (out_filepath) { # Get the data from ScienceBase
+  
+  mendota_file <- file.path(out_filepath)
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
```

Comment:
Not a request for a change in this PR, but something to think about for the future: writing this function so that it accepts the ScienceBase item id and the name of the ScienceBase file could make this function more reusable in case you wanted to download other files from ScienceBase.